platform-emf/src/main/resources/com/softicar/platform/emf/emf-form-style.css

```
@@ -1,8 +1,12 @@
 
 .EmfForm {
+	width: 100%; 
+}
+
+.EmfFormContainer {
 	display: grid;
 	flex-grow: 1;
-	min-width: 320px;
+	min-width: 320px; 
```

why adding a trailing whitespace here?
```suggestion
	min-width: 320px;
```